pages/index.html

```diff
@@ -190,7 +190,11 @@ <h4>Related projects:</h4>
 {%- for cat in site.data.collabcats -%}
   {%- for proj in site.data.collaborations[cat.id] -%}
     {%- capture projtxt -%}
-      [{{ proj.name }}]({{ proj.url }})
+      {%- if proj.short %}
```

Comment:
Oops, this broke the page; the `-` is important, it removes all whitespace. Fix incoming in #960.